### DISCUSSION TITLE
相關係數作業

### DISCUSSION THREAD

**Discussion by Will150143** [ChiuAlfredo/options_day_data#38] [研究]
##  1.不同計算方式(注意四張圖的縱橫軸的公式)但同時間
![螢幕擷取畫面 2024-07-12 184811](https://github.com/user-attachments/assets/3071a628-81d6-48e8-bbb9-dbf3a51d5c3c)
*圖片裡的C改成期貨

**Comment by TsuYiii** [comment]
針對每一價位做出相關係數與分佈圖，再取算術平均

**Comment by TsuYiii** [comment]
# 結果

## $C S$
* Correlation= 0.984446804

## $\delta C \delta S$
* Correlation=0.507468642

## $\frac{\delta C}{C}$ $\frac{\delta S}{S}$ 
* Correlation=0.509368959

## $ln(\frac{C}{C}) ln(\frac{S}{S})$
* Correlation=0.508425028

**Comment by TsuYiii** [comment]
<img width="361" alt="image" src="https://github.com/user-attachments/assets/1cab11e7-936b-43d2-b5b7-4ee923df56cf">

**Comment by TsuYiii** [comment]
<img width="362" alt="image" src="https://github.com/user-attachments/assets/240b0b9c-cf1d-4d09-a469-203e0afa8117">

**Comment by TsuYiii** [comment]
<img width="361" alt="image" src="https://github.com/user-attachments/assets/e2c4853a-3818-4522-87e3-0fd9b75da629">

**Comment by TsuYiii** [comment]
<img width="361" alt="image" src="https://github.com/user-attachments/assets/4899b5c6-fb80-44a9-94f9-0583a8ffaeef">

**Comment by TsuYiii** [comment]
https://mailntustedutw-my.sharepoint.com/:x:/r/personal/b11033003_ms_ntust_edu_tw/_layouts/15/Doc.aspx?sourcedoc=%7B5FB28616-73E2-4369-A088-576030260867%7D&file=%E7%9B%B8%E9%97%9C%E4%BF%82%E6%95%B8%E6%95%A3%E5%B8%83%E5%9C%96%E4%BD%9C%E6%A5%AD.xlsx&action=default&mobileredirect=true&wdsle=0
